src/safeds/ml/nn/_input_conversion_image.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import sys
+from typing import TYPE_CHECKING, Any
+
+from safeds._utils import _structural_hash
+from safeds.data.image.containers import ImageList
+from safeds.data.image.containers._single_size_image_list import _SingleSizeImageList
+from safeds.data.labeled.containers import ImageDataset
+from safeds.data.labeled.containers._image_dataset import _ColumnAsTensor, _TableAsTensor
+
+if TYPE_CHECKING:
+    from safeds.data.image.typing import ImageSize
+    from safeds.data.tabular.transformation import OneHotEncoder
+
+from safeds.ml.nn import InputConversion
+
+
+class InputConversionImage(InputConversion[ImageDataset, ImageList]):
+    """The input conversion for a neural network, defines the input parameters for the neural network."""
+
+    def __init__(self, image_size: ImageSize) -> None:
+        """
```

Comment:
Can we get the ``ìmage_size``` information from input_data in the Imageset?
in my PR I try to get rid of most arguments in the init function, so the user has a more easier handle.

Reply:
I need the `image_size` for the creation of the 2d layers. Currently, the pytorch model is created in the init of the model class. Therefore, the `image_size` needs to be set before the model is initialized.